templates/wildfly.systemd.service.epp

```diff
@@ -9,7 +9,7 @@ User=<%= $wildfly::user %>
 Group=<%= $wildfly::group %>
 EnvironmentFile=/etc/wildfly/wildfly.conf
 LimitNOFILE=102642
-ExecStart=<%= $wildfly::dirname %>/bin/launch.sh ${WILDFLY_MODE} ${WILDFLY_CONFIG}
+ExecStart=<%= $wildfly::dirname %>/bin/launch.sh ${WILDFLY_MODE} ${WILDFLY_CONFIG} -P ${WILDFLY_PROPERTIES}
```

Comment:
Quotes should probably be present here for the same reason:

```suggestion
ExecStart=<%= $wildfly::dirname %>/bin/launch.sh ${WILDFLY_MODE} "${WILDFLY_CONFIG}" -P "${WILDFLY_PROPERTIES}"
```

I have not checked for WILDFLY_MODE.